examples/with-userbase/components/todo-form/index.js

```diff
@@ -0,0 +1,133 @@
+import { useState, useEffect } from 'react'
+import userbase from 'userbase-js'
+
+function Todo({ name, done, toggleComplete, deleteTodo }) {
+  return (
+    <li className="my-4">
+      <div className="flex items-center">
+        <span className={done ? 'text-gray-500' : ''}>{name}</span>
+        <button
+          type="button"
+          className="mx-4 p-1 rounded bg-purple-400 text-white font-bold"
+          onClick={e => {
+            e.preventDefault()
+            toggleComplete()
+          }}
+        >
+          {done ? 'not done' : 'done'}
+        </button>
+        <button
+          type="button"
+          onClick={e => {
+            e.preventDefault()
+            deleteTodo()
+          }}
+          className=" p-1 bg-red-500 text-white rounded font-bold"
+        >
+          delete
+        </button>
+      </div>
+    </li>
+  )
+}
+
+function TodoForm() {
+  const [newTodo, setNewTodo] = useState()
```

Comment:
```suggestion
  const [newTodo, setNewTodo] = useState('')
```
This needs to start as a string or else it's switching the input from controlled to uncontrolled which is a react warning

<img width="1205" alt="Screen Shot 2020-04-23 at 16 20 29" src="https://user-images.githubusercontent.com/22380829/80150652-5d8b3880-857e-11ea-8f62-c53b963d54c4.png">